src/pure-utils/actionToPath.js

```diff
@@ -32,22 +29,29 @@ export default (
   return search ? `${path}?${search}` : path
 }
 
-const _payloadToParams = (route: RouteObject, params: Payload = {}): Params =>
+const _payloadToParams = (route: Route, params: Payload = {}): Params =>
   Object.keys(params).reduce((sluggifedParams, key) => {
-    if (typeof params[key] !== 'undefined') {
-      if (typeof params[key] === 'number') {
-        sluggifedParams[key] = params[key]
-      }
-      else if (route.capitalizedWords === true) {
-        sluggifedParams[key] = params[key].replace(/ /g, '-').toLowerCase()
-      }
-      else if (typeof route.toPath === 'function') {
-        sluggifedParams[key] = route.toPath(params[key], key)
-      }
-      else if (typeof params[key] === 'string') {
-        sluggifedParams[key] = params[key]
-      }
-    }
-
+    const segment = params[key]
```

Comment:
might consider `Object.entries` if both the key and value are important -- avoids reference to object

Reply:
Also Im not sure "segment" is the clearest name, but perhaps Im just not familiar with it's use in reference to param values.